### PR TITLE
Thwomper's Synthetic Uniforms(THWOMP SHRAMALERT) & Synth Vendor QOL

### PR DESCRIPTION
**Pull request by QuickLode**
# About the pull request

Incorporates Thwomper's Synthetic Uniforms & Vest into the game.
(I tried to be creative with the lore but I'm not paid by the hour man! that UA laborer is.)

Synthetic Vendor QOL making it easier to find and use.

Slight moving of stuff, vestigial stuff moved too. 

# Explain why it's good for the game
THWOMP SHRAMALERT

Gives a UA Colony/Shipside theme.

QOL makes things easier to use.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.
tested and everything appears to be working well(tahts how i got the screenshot)
https://cdn.discordapp.com/attachments/777257424590995466/1126429970244325406/image.png
</details>

# Changelog
:cl: QuickLoad, Thwomper
imageadd: Synthetic Uniforms & Vest with United Americas flair and classic jumpsuits which remind you of life on the Frontier. Sprited by THE THWOMPA himself!
qol: Synthetic Vendor is more organized. Removes scrub cap, adds shoe.
/:cl:

### CODE REVIEW

**Review by sleepynecrons — CHANGES_REQUESTED** [review]
all of the outfits look great except for cargo_light. the shading is very flat, lacks contrast and needs a proper outline like the rest of the outfits
image
the jacket tied around the waist is also at inconsistent heights and there is a random pixel on the east/west sprites

**Review by nauticall — APPROVED** [review]
cargo light looks good to me but I'll wait for sleepy to approve too

**Review by nauticall — CHANGES_REQUESTED** [review]
mb the shading on the shirt still needs work. shading should be darker round the top and more outlined

**Comment by thwompa** [comment]
should be GOOD2GO(tm patent pending)

**Review by nauticall — APPROVED** [review]
now THAT is spriting :100:

File: code/modules/clothing/under/marine_uniform.dm
Comment (nauticall) [inline code/modules/clothing/under/marine_uniform.dm:716]: add \improper before these (same with any non-proper names that begin with a capital letter) and turn it to "UA [color] utility uniform"

`\improper UA gray utilities` results in "That's a UA gray utility uniform." whereas without the `\improper` it just puts "That's UA gray utility uniform."

**Review by nauticall — CHANGES_REQUESTED** [review]

File: code/modules/clothing/under/marine_uniform.dm
Comment (nauticall) [reply code/modules/clothing/under/marine_uniform.dm:716]: once that's done I'll merge it....

Comment (nauticall) [reply code/modules/clothing/under/marine_uniform.dm:716]:
```suggestion
	name = "\improper UA brown utility uniform"
```

File: code/modules/clothing/under/marine_uniform.dm
Comment (nauticall) [inline code/modules/clothing/under/marine_uniform.dm:708]:
```suggestion
	name = "\improper UA gray utility uniform"
```

**Review by nauticall — APPROVED** [review]
gouda 🧀

**Pull request merged**
